AirCasting/Graph/AirCastingGraph.swift

```diff
@@ -103,7 +103,7 @@ class AirCastingGraph: UIView {
         // Line styling
         dataSet.drawCirclesEnabled = false
         dataSet.setColor(UIColor(.white))
-        dataSet.mode = .horizontalBezier
+        dataSet.mode = .cubicBezier
```

Comment:
Didn't Michael prefer the "horizontalBezier" at some point?

Reply:
I am not sure really, we will see then :)